### PR TITLE
chapter: startingcommands

### PR DESCRIPTION
**Pull request by ghost**
I've separated the commits into typo fixing and enhancements. Not sure if these enhancements are desired, but if so, I'd be happy to go through the next chapters as well. Should I further break down commits?

### CODE REVIEW

**Review by jasonbraganza — APPROVED** [review]
👍

**Review by jasonbraganza — APPROVED** [review]

**Review by jasonbraganza — APPROVED** [review]

**Comment by jasonbraganza** [comment]
No. This is granular enough. Thank you so much for all you did :)

**Review by kushaldas — CHANGES_REQUESTED** [review]
@Schubisu one small change requested.

File: docs/startingcommands.rst [hunk 1]
```diff
@@ -1,24 +1,24 @@
 Shell commands
 ===============
 
-Linux shell or the terminal is the lifeline of the developers, and of any
+In Gnu/Linux the shell (or terminal) is the lifeline of the developer, and of any
```
Comment (kushaldas) [inline docs/startingcommands.rst]: Please keep the OS name as *Linux*.

Comment (ghost) [reply docs/startingcommands.rst]: I've changed the wording accordingly

**Review by kushaldas — APPROVED** [review]
Thank you :unicorn: :rainbow:

**Pull request merged**